pydra/engine/tests/data_tests/loading.py

```diff
@@ -2,7 +2,7 @@
 
 
 def loading(filename):
-    with open(filename, "r") as f:
+    with open(filename) as f:
```

Comment:
Explicit is generally better than implicit. I would revert this one:

```suggestion
    with open(filename, "r") as f:
```

Reply:
I feel this one is pretty obvious.

I also don't see `d.get(key, None)` everywhere for example 😄 :trollface:

Reply:
I use often `with open(filename, "r") as f`, but I agree that ` with open(filename) as f` is pretty standard and looks good to me